src/PointwiseFunctions/AnalyticSolutions/Tags.hpp

```diff
@@ -50,6 +50,7 @@ struct AnalyticSolution : AnalyticSolutionBase, db::SimpleTag {
   static std::string name() noexcept { return "AnalyticSolution"; }
   using type = SolutionType;
   using option_tags = tmpl::list<::OptionTags::AnalyticSolution<SolutionType>>;
+  template <typename Metavariables>
```

Comment:
These ones didn't get the extra newlines the others got. Any reason to leave them out?

Reply:
Hmm, seems we are now consistently having the newline. I'll add it. Good catch!